tests/unit/project_api/test_project_api.py

```diff
@@ -219,3 +236,150 @@ def test_get_workgroup_review_items(self, reviewer_id, content_id, review_items,
 
             self.assertEqual(result, expected_result)
             patched_get_review_items.assert_called_once_with('group_id', content_id)
+
+    def assert_project_data(self, project_data, expected_values):
+        attrs_to_test = [
+            "id", "url", "created", "modified", "course_id", "content_id", "organization", "workgroups"
+        ]
+        for attr in attrs_to_test:
+            self.assertEqual(getattr(project_data, attr), expected_values[attr])
+
+    def test_get_project_details(self):
+        calls_and_results = {
+            (PROJECTS_API, 1): canned_responses.Projects.project1,
+            (PROJECTS_API, 2): canned_responses.Projects.project2
+        }
+
+        expected_calls = [
+            mock.call(GET, (PROJECTS_API, 1), no_trailing_slash=True),
+            mock.call(GET, (PROJECTS_API, 2), no_trailing_slash=True)
+        ]
+
+        with self._patch_send_request(calls_and_results) as patched_send_request:
+            project1 = self.project_api.get_project_details(1)
+            project2 = self.project_api.get_project_details(2)
+
+            self.assertEqual(patched_send_request.mock_calls, expected_calls)
+
+        self.assert_project_data(project1, canned_responses.Projects.project1)
+        self.assert_project_data(project2, canned_responses.Projects.project2)
+
+    @ddt.data(
+        ('course1', 'content1'),
+        ('course2', 'content2'),
+    )
+    @ddt.unpack
+    def test_get_project_by_content_id(self, course_id, content_id):
+        expected_parameters = {
+            'course_id': course_id,
+            'content_id': content_id
+        }
+        calls_and_results = {(PROJECTS_API,): [canned_responses.Projects.project1]}
+
+        with self._patch_send_request(calls_and_results) as patched_send_request:
+            project = self.project_api.get_project_by_content_id(course_id, content_id)
+            self.assert_project_data(project, canned_responses.Projects.project1)
+
+            patched_send_request.assert_called_once_with(GET, (PROJECTS_API,), query_params=expected_parameters)
+
+    def test_get_project_by_content_id_fail_if_more_than_one(self):
+        calls_and_results = {
+            (PROJECTS_API,): [canned_responses.Projects.project1, canned_responses.Projects.project2]
+        }
+        with self._patch_send_request(calls_and_results), \
```

Comment:
Nit: I think this is easier to read if you put it on one line and remove the `\` continuation